### ISSUE TITLE
Replace use of file 'source' and 'format' fields with replacement fields

### ISSUE THREAD

**Issue by dsotirho-ucsc** [DataBiosphere/data-browser#1898]
https://github.com/DataBiosphere/azul/issues/3111 will deprecate the following fields. These deprecated fields will remain in the response until their use has been discontinued.
(After 3111 has been deployed to production) please update the Data Browser to make use of the replacement fields instead.

```
hits[].files[].matrix_cell_count (superseded by matrixCellCount)
hits[].files[].source (superseded by fileSource)

hits[].fileTypeSummaries[].fileType (superseded by format)
hits[].fileTypeSummaries[].source (superseded by fileSource)

hits[].projects[].contributorMatrices...matrix_cell_count (superseded by matrixCellCount)
hits[].projects[].contributorMatrices...source (superseded by fileSource)
hits[].projects[].matrices...matrix_cell_count (superseded by matrixCellCount)
hits[].projects[].matrices...source (superseded by fileSource)
```

**Comment by danielsotirhos** [comment]
Blocked by https://github.com/DataBiosphere/azul/issues/3111

**Comment by NoopDog** [comment]
Thanks for the update @danielsotirhos

**Comment by NoopDog** [comment]
@danielsotirhos @theathorn We think we have removed references to these fields. You should be unblocked to remove these from the Azul response and test on dev. Thanks! D
